PayPalWebPayments/src/main/java/com/paypal/android/paypalwebpayments/PayPalWebCheckoutClient.kt

```diff
@@ -40,36 +42,29 @@ class PayPalWebCheckoutClient internal constructor(
         urlScheme: String
     ) : this(
         activity,
-        configuration,
         AnalyticsService(activity.applicationContext, configuration),
-        BrowserSwitchClient(),
-        BrowserSwitchHelper(urlScheme)
+        PayPalWebLauncher(urlScheme, configuration),
+        PayPalWebCheckoutVaultExperienceContext(
+            urlScheme,
+            domain = VAULT_HOST,
+            returnUrlPath = VAULT_RETURN_URL_PATH,
+            cancelUrlPath = VAULT_CANCEL_URL_PATH
+        )
     )
 
-    private val exceptionHandler = CoreCoroutineExceptionHandler {
-        deliverFailure(it)
-    }
-
-    private var browserSwitchResult: BrowserSwitchResult? = null
-
-    private var orderId: String? = null
```

Comment:
I decided to remove these internal state properties. We should avoid having mutable state in the clients whenever possible. It also makes the tests much simpler to write. We don't have these on BT so I don't think it'll be problematic.